generator/src/init-sources.ts

```diff
@@ -32,7 +32,7 @@ export class InitSources {
     public static readonly PREFIX_PACKAGES_EXTENSIONS = '@che-';
 
     public static readonly DEFAULT_EXTENSIONS_URI =
-        'https://raw.githubusercontent.com/eclipse-che/che-theia/master/che-theia-init-sources.yml';
+        'https://raw.githubusercontent.com/eclipse-che/che-theia/master-main/che-theia-init-sources.yml';
```

Comment:
This change is to pass the tests. I'll update that to `main` after merging this PR.

Reply:
https://github.com/eclipse-che/che-theia/pull/1149